daemon/cmd/daemon_main.go

```diff
@@ -1623,6 +1624,7 @@ type daemonParams struct {
 	HealthAPISpec        *healthApi.Spec
 	ServiceCache         *k8s.ServiceCache
 	ClusterMesh          *clustermesh.ClusterMesh
+	GARPEventProcessor   garp.EventProcessor `optional:"true"`
```

Comment:
I don't see anything using this in daemon so this can be removed. Was this added here just to enforce the construction of `GARPEventProcessor`? See comment about using `Invoke` in `events.go` on how this should be done.